signalr-node-server/app.js

```diff
@@ -4,28 +4,27 @@ const http = require('http').createServer(app);
 const signalR = require('./signalr')(http);
 const port = 3000;
 
-const chat = signalR.mapHub('/chat');
+const chat = signalR.mapHub('/chatHub');
 const clock = signalR.mapHub('/clock');
 
 setInterval(() => {
     clock.clients.all.send('tick', Date.now());
 }, 1000);
 
-chat.on('connect', (id) => {
+chat.on('connect', id => {
     console.log(`${id} connected`);
 });
 
-chat.on('disconnect', (id) => {
+chat.on('disconnect', id => {
     console.log(`${id} disconnected`);
 });
 
-chat.on('send', (message) => {
-    chat.clients.all.send('send', message);
+chat.on('sendmessage', (user, message) => {
+    chat.clients.all.send('receivemessage', user, message);
```

Comment:
changed this to lowercase events for consistency with the "sendmessage" toLowerCase function in signalr.js